src/algebra/lie_algebra.lean

```diff
@@ -409,39 +441,120 @@ instance lie_subalgebra_lie_algebra (L' : lie_subalgebra R L) :
     @lie_algebra R L' _ (lie_subalgebra_lie_ring _ _ _) :=
 { lie_smul := by { intros, apply set_coe.ext, apply lie_smul } }
 
+@[simp] lemma lie_subalgebra.mem_coe {L' : lie_subalgebra R L} {x : L} :
+  x ∈ (L' : set L) ↔ x ∈ L' := iff.rfl
+
+@[simp, norm_cast] lemma lie_subalgebra_coe_bracket (L' : lie_subalgebra R L) (x y : L') :
+  (↑⁅x, y⁆ : L) = ⁅↑x, ↑y⁆ := rfl
+
+@[ext] lemma lie_subalgebra_ext (L₁' L₂' : lie_subalgebra R L) (h : (L₁' : set L) = L₂') :
```

Comment:
```suggestion
@[ext] lemma lie_subalgebra.ext (L₁' L₂' : lie_subalgebra R L) (h : (L₁' : set L) = L₂') :
```
Don't you want to state the hyp `h` as `\forall x, x \in L1 \iff x \in L2`?
Also, whatever you choose, please provide an `ext_iff` as well.

Reply:
Type of `h` changed to suggested form and `ext_iff` added.